web/app/js/components/Tap.jsx

```diff
@@ -241,6 +241,21 @@ class Tap extends React.Component {
     this.setState({ tapIsClosing: true });
   }
 
+  handleTapClear = () => {
+    this.resetTapResults();
+    this.setState({
+      query: emptyTapQuery()
+    });
```

Comment:
`resetTapResults` already sets `query: emptyTapQuery()` into state. So maybe this extra call to `setState` is unnecessary?

Reply:
ah yeah! good call!